site.cfg.example

```diff
@@ -52,6 +52,26 @@
 #       True) to tell numpy.distutils to prefer static libraries (.a) over
 #       shared libraries (.so). It is turned off by default.
 #           search_static_first = false
+#
+#   runtime_library_dirs
+#       List of directories that contains the libraries that should be 
+#       used at runtime, thereby disregarding the LD_LIBRARY_PATH variable.
```

Comment:
it's more "removing the need to use" rather then "disregarding" right? One can still use a combination of both ``runtime_library_dirs`` and ``LD_LIBRARY_PATH`` if desired.